config/config.yml

```
@@ -0,0 +1,128 @@
+# salmon quantification parameters
+quant:
+    # Salmon library type (Default: U)
+    salmon_libtype: "U"
+
+# This section defines the pyDESeq2 plot and data handling parameters
+deseq2:
+    # normalization fit type, must be 'parametric' or 'mean'
+    fit_type: "mean"
+    # the "design factors" are the confounding variables to be adjusted fr
+    # during normalization. They must be given in the configuration (samples.csv).
+    design_factors: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)